ModelicaTest/Fluid/TestComponents/Machines/TestLinearPower.mo

```diff
@@ -4,6 +4,6 @@ model TestLinearPower
   Modelica.Units.SI.Power p;
 equation
   p = Modelica.Fluid.Machines.BaseClasses.PumpCharacteristics.linearPower(V_flow_nominal={1,2}, W_nominal={3,5}, V_flow=11);
-  assert(p >= 23 and p <= 23, "Error in linearPower");
+  assert(abs(p - 23) < 1e-9, "Error in linearPower");
```

Comment:
```suggestion
  assert(Modelica.Math.isEqual(p, 23.0, 1e-9), "Error in linearPower");
```

Reply:
Yeah, I mean, thanks to inlining it's exactly the same, only with some added abstraction. Does it really matter?